automatic/screencloud/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,26 @@
+$ErrorActionPreference = 'Stop'
+
+[array]$key = Get-UninstallRegistryKey -SoftwareName 'ScreenCloud'
+
+if($($key.Count) -eq 1){
+    $key | % {
+        $file = "$($_.UninstallString)"
+
+        $packageArgs = @{
+            packageName    = 'screencloud'
+            fileType       = 'msi'
+            silentArgs     = "$($_.PSChildName) /qn /norestart"
+            validExitCodes = @(0)
+            file           = ''
+        }
+
+        Uninstall-ChocolateyPackage @packageArgs
+    }
+} elseif ($($key.Count) -eq 0){
```

Comment:
You misunderstood @AdmiringWorm here:

* `if ($($key.Count) -eq 0)` should be reverted back to `if ($key.Count -eq 0)`
* `"$key.Count matches found!`" should be changed to `"$($key.Count) matches found!"`